data/api/client-server/definitions/room_event_filter.yaml

```diff
@@ -28,15 +28,17 @@ allOf:
       description: |-
         If `true`, enables lazy-loading of membership events. See
         [Lazy-loading room members](/client-server-api/#lazy-loading-room-members)
-        for more information. Defaults to `false`.
+        for more information. Defaults to `false`. Only applies to the `/messages`
```

Comment:
It should also apply to the `/context` endpoint, as listed in the "lazy-loading room members" section. I believe it would be better to only list exceptions here.